Automattic/wp-calypso
# New Checkout: add Tracks event when checkout modal is closed by cancelling.

This… adds a Tracks event when the checkout delete product modal is closed by cancelling :)

**To test:**
- run `localStorage.setItem( 'debug', 'calypso:analytics*' );` in your console and refresh the page
- add a plan to your cart and visit new Checkout with `?flags=composite-checkout-force`
- from the third step, click the delete item trash can
- cancel the modal
- verify that the `calypso_checkout_composite_cancel_delete_product` event fires
